docs/index.html

```diff
@@ -631,6 +631,38 @@ <h2 id="setreturnfocus-heading">setReturnFocus option</h2>
       </p>
     </div>
 
+    <div id="demo-setreturnfocus-function">
+      <h2 id="setreturnfocus-function-heading">setReturnFocus as function</h2>
+      <p>
+        As the function is executed in the process of deactivation, it allows you to dynamically specify which element should be focused after deactivation.
+      </p>
```

Comment:
Adding a second paragraph here, like "Once the trap is activated, you can still click on one of the 3 deactivate buttons below in order to deactivate the trap." would be helpful.